instrumentation/net/http/httptrace/otelhttptrace/clienttrace.go

```diff
@@ -262,7 +278,7 @@ func (ct *clientTracer) span(hook string) trace.Span {
 }
 
 func (ct *clientTracer) getConn(host string) {
-	ct.start("http.getconn", "http.getconn", semconv.NetHostName(host))
+	ct.start("http.getconn", "http.getconn", attribute.Key("net.host.name").String(host))
```

Comment:
Don't change this please.

Reply:
Whoops, sorry about that, that was changed as it was being called out as a deprecated attribute in semconv v1.24.0 -- I've restored the original call.